UVtools.Core/Operations/OperationPixelArithmetic.cs

```diff
@@ -57,7 +58,10 @@ public StringMatrix(string text)
         private byte _patternGenBrightness = 128;
         private byte _patternGenInfillThickness = 10;
         private byte _patternGenInfillSpacing = 20;
-        
+        private bool _preserveVoids = true;
+        private int _minNoiseOffset = -128;
+        private int _maxNoiseOffset = 128;
+
```

Comment:
Use the smallest type possible: **short**

Reply:
Adjusted, note this will be immediately cast to int on usage in the random number generator (only usage).